src/microbe_stage/MicrobeStage.cs

```diff
@@ -671,7 +671,12 @@ public void OnReturnFromEditor()
         {
             daughter.BecomeFullyGrownMulticellularColony();
 
-            // TODO: add more extra offset between the player and the divided cell
+            // Back the daughter away so the player isn't stuck inside
+            while (daughter.Colony!.ColonyMembers.Any(member =>
```

Comment:
I think this logic should probably go in Microbe.Divide where I already did custom separation logic for colonies.

Reply:
That won't work because this method calls BecomeFullyGrownMulticellularColony() is called after divide.

Reply:
What about the comment few lines above: `// This is done first to ensure that the player colony is still intact for spawn separation calculation`

My new code for colony split separation definitely gets called, after all if you compare the RC to 0.5.8 the separation is obviously different.

So, really I'd like you to improve this bit of code (found in Microbe.Divide):
```
            // When in a colony we approximate a much higher separation distance
            var colonyRadius = separation.x;

            foreach (var colonyMember in Colony.ColonyMembers)
            {
                if (colonyMember == this)
                    continue;

                var radius = colonyMember.Radius + Constants.COLONY_DIVIDE_EXTRA_DAUGHTER_OFFSET;

                // TODO: switch this to something else if this is too slow for large colonies
                var positionInColony = colonyMember.GlobalTransform.origin - currentPosition;

                var outerRadius = Math.Max(Math.Abs(positionInColony.x) + radius,
                    Math.Abs(positionInColony.z) + radius);

                if (outerRadius > colonyRadius)
                    colonyRadius = outerRadius;
            }

            separation = new Vector3(colonyRadius + Constants.COLONY_DIVIDE_EXTRA_DAUGHTER_OFFSET, 0, 0);
```
and preferably you would come up with an algorithm that isn't a bunch of steps * the number of colony members of complexity.